core/src/main/scala/scalafix/rewrite/ProcedureSyntax.scala

```diff
@@ -36,5 +36,6 @@ class ProcedureSyntax[T]() extends Rewrite[T] {
 }
 
 object ProcedureSyntax {
+  val default: ScalafixRewrite = apply()
```

Comment:
This is subtle.  I think it says that calling apply through default fixes an equality bug.  If so, I would add a comment here to prevent someone from inlining it in the future.